app/components/ExternalLinksBlock/components/ExternalLink/ExternalLink.jsx

```diff
@@ -30,6 +30,11 @@ export default function ExternalLink({ className, isExternalLink, isSocialLink,
     iconClasses = `fa ${icon} fa-fw`;
     linkLabel = path;
     linkProps['aria-label'] = socialIcons[site].ariaLabel;
+    if (site === 'wordpress') {
+      linkProps.href = `https://${path}.${site}`;
+    } else {
+      linkProps.href = `https://${site}${path}`;
+    }
     const isValidLink = !!icon && linkProps.href.substring(0, 8) === 'https://';
```

Comment:
One thing to note is that the `linkProps.href.substring(0, 8) === 'https://'` becomes irrelevant, as the previous `if-else` code block always creates `https://` links.